Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 
 Compiler Features:
  * Commandline Interface: Normalize paths specified on the command line and make them relative for files located inside base path.
+ * Commandline Interface: Report optimizer options as invalid in Standard JSON and linker modes instead of ignoring them.
```

Comment:
It's rather a bugfix, isn't it?

Reply:
Not sure we consider missing checks a bug. So far I've been treating it as a missing feature, both when labeling issues and when adding changelog entries in the past. Changing it now would be a bit inconsistent.

Reply:
To me a missing check especially like this one, i.e. not reporting an error for something that is entirely ignored and thereby misleading, is rather clearly a bugfix...

Reply:
If you think of it as a feature, i.e. the old behaviour was not a bug, but we actively change things to new behavior as a new feature, we'd strictly speaking even have to think about whether this is breaking :-). (I'm not seriously saying we should, but if it's a feature that results in compiler invocations that used to work to now stop working, which is the case here, then it's pretty much a breaking feature - thus I'd much rather say this check should always have been there and it missing was just a bug that we now fix)

Also quickly looking through the changelog I don't see anything like adding a missing check being files as feature except one (``Command Line Interface: Only accept library addresses with a prefix of ``0x`` in ``--libraries`` commandline option.``, which also doesn't really strike me as feature). Cases similar to this one here like ``Commandline interface: Fix standard input bypassing allowed path checks.`` are in fact files as bugfixes.

That being said, it's not really important to me, and I expected this not to be intentional and just an accident. I'd myself not hesitate to file it under Bugfixes, but I'm also fine with keeping it, even though I don't see a "feature" here myself :-).

Reply:
True, technically it's breaking but in the past @chriseth allowed them in non-breaking releases anyway. But ok, I'm happy treating it as a bugfix. I just thought that the old behavior was intentional because it was so common in `CommandLineInterface`. Changelog entry changed.

> Also quickly looking through the changelog I don't see anything like adding a missing check being files as feature except one

There were more, see for example the one from this release that I now made a bugfix as well (see the diff). It's no like there were tons of them but still, I think I could list at least a few.